apps/posts/src/views/PostAnalytics/PostAnalytics.tsx

```
@@ -0,0 +1,50 @@
+import PostAnalyticsLayout from './layout/PostAnalyticsLayout';
+import PostAnalyticsView from './components/PostAnalyticsView';
+import {Breadcrumb, BreadcrumbItem, BreadcrumbLink, BreadcrumbList, BreadcrumbPage, BreadcrumbSeparator, H1, ViewHeader} from '@tryghost/shade';
+import {useNavigate, useParams} from '@tryghost/admin-x-framework';
+
+interface postAnalyticsProps {};
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)